### ISSUE TITLE
k8s: custom pod names

### ISSUE THREAD

**Issue by dragoslav** [magneticio/vamp#951]
Vamp manages deployments and pod names can be set via dialects or something similar.
Example use case: "we use the pod names to identify where the logs come from".

**Comment by thecodeassassin** [comment]
Currently the naming seems to be random. In standard kubernetes the pod name comes from tbe deployment. Does vamp use kubernetes deployments to schedule the pods?

**Comment by dragoslav** [comment]
Yes, Vamp uses k8s deployments, but also depending on the name, we may hash them. Default names are based on Vamp deployments/clusters/services.
